cmd/brew-test-bot.rb

```diff
@@ -533,12 +533,17 @@ def download
           "git", "-C", @tap.path.to_s,
                  "log", "-1", "--format=%h (%s)", "origin/master"
         ).strip
-        puts "#{@tap} origin/master #{tap_origin_master_revision}"
         tap_revision = Utils.popen_read(
           "git", "-C", @tap.path.to_s,
                  "log", "-1", "--format=%h (%s)"
         ).strip
-        puts "#{@tap} HEAD #{tap_revision}"
+        puts <<~EOS
+          \nTesting tap #{@tap} with:
```

Comment:
Handle if `@tap` isn't set.

Reply:
You mean `nil` or `blank`? If `nil`, won't it be taken care of by the `if @tap`?

Reply:
@MikeMcQuaid or do you mean with an `else` clause?

Reply:
Sorry, I missed that it was indented in there. I think ideally there'd be consistent output of this stuff (e.g. `diff_start_sha1`) regardless of whether a `@tap` is set.

How about pulling it out the `if` and doing something like `"Testing#{" tap #{@tap}" if tap.present?} with:"`?

After that I think we're good to go here 👍

Reply:
sounds good, I'll update the PR now :)